packages/core/components/Puck/index.tsx

```diff
@@ -410,8 +411,8 @@ export function Puck({
                     <div
                       className="puck-root"
                       style={{
-                        background: "white",
                         border: "1px solid var(--puck-color-grey-8)",
+                        boxShadow: "0px 0px 0px 32px var(--puck-color-grey-10)",
```

Comment:
Not sure I understand the reason for this change? Can you share a screenshot of the original issue?

This fix causes and additional white border in Safari and Chrome due to the zoom factor (FF doesn't respect zoom per #15)

<img width="1624" alt="image" src="https://github.com/measuredco/puck/assets/985961/5d7a76e1-fe80-4939-ba50-cfed3abaf4be">

Reply:
Ah yeah, it definitely isn't the ideal solution. Reason was setting background on the frame would cause the preview root to either:
1. Inherit from the frame if set to inherit
2. If no BG is set on preview root, the frame BG is all that would show rather than the background for the document body

![image](https://github.com/measuredco/puck/assets/42180996/9b69cc00-bf77-478d-9fca-208a857de97b)
The example here being the 2nd case with the frame BG taking over.

This was my primary reason for asking about portals in #106. I could also imagine a fix involving pseudo-elements, though would need to move some styles out of inline styles.